openstef_dbc/data_interface.py

```diff
@@ -223,33 +260,34 @@ def check_influx_available(self):
 
     def exec_sql_query(self, query: str, params: dict = None):
         try:
-            connection = self.mysql_engine.connect()
+            connection = self.sql_engine.connect()
             if params is None:
                 params = {}
             cursor = connection.execute(query, **params)
             connection.close()
             if cursor.cursor is not None:
                 return pd.DataFrame(cursor.fetchall())
         except sqlalchemy.exc.OperationalError as e:
-            self.logger.error("Lost connection to MySQL database", exc_info=e)
+            self.logger.error("Lost connection to {} database".format(self.sql_db_type.upper()), exc_info=e)
             raise
         except sqlalchemy.exc.ProgrammingError as e:
             self.logger.error(
                 "Error occured during executing query", query=query, exc_info=e
             )
             raise
         except sqlalchemy.exc.DatabaseError as e:
-            self.logger.error("Can't connect to MySQL database", exc_info=e)
+            self.logger.error("Can't connect to {} database".format(self.sql_db_type.upper()), exc_info=e)
             raise
 
     def exec_sql_write(self, statement: str, params: dict = None) -> None:
         try:
-            with self.mysql_engine.connect() as connection:
+            with self.sql_engine.connect() as connection:
                 response = connection.execute(statement, params=params)
 
                 self.logger.info(
-                    "Added {} new systems to the systems table in the MySQL database".format(
-                        response.rowcount
+                    "Added {} new systems to the systems table in the {} database".format(
```

Comment:
Please write this as an f-string. Also here upper is called, better to define sql_db_type as an enum and use that.

Reply:
yes done